connector_search_engine/models/se_binding.py

```diff
@@ -190,11 +190,11 @@ def synchronize(self):
         # and to do the right action as we are in a transaction.
         export_ids = []
         delete_ids = []
-        for work in self._work_by_index():
+        for work in self.sudo()._work_by_index():
```

Comment:
What is your case when you need a sudo ?

Reply:
@simahawk

Reply:
@sebastienbeau for the same reason described here https://github.com/OCA/search-engine/blob/14.0/connector_search_engine/models/se_binding.py#L134-L139

Reply:
Ok